### ISSUE TITLE
Will Internet Exporer 11 be supported?

### ISSUE THREAD

**Issue by sugizon** [remarkablemark/html-react-parser#222]
## Description
Will Internet Exporer 11 be supported?
I think the following code in index.mjs is not supported by Internet Explorer 11.
`export var { domToReact, htmlToDOM, attributesToProps } = HTMLReactParser;`
Is it possible to support it?

## Environment

- Version: v1.2.0 later
- Platform: Windows 10
- Browser: Internet Explorer 11

**Comment by remarkablemark** [comment]
Thanks for opening this issue @sugizon

This library will support IE11 until its sunset date of August 17, 2021 (see [source](https://techcommunity.microsoft.com/t5/microsoft-365-blog/microsoft-365-apps-say-farewell-to-internet-explorer-11-and/ba-p/1591666)).

It seems like I introduced a bug in #220. I will open a PR to fix it.

**Comment by remarkablemark** [comment]
Published [v1.2.4](https://github.com/remarkablemark/html-react-parser/releases/tag/v1.2.4):

[npm](https://www.npmjs.com/package/html-react-parser):

```sh
npm i html-react-parser@1.2.4
```

[Yarn](https://yarnpkg.com/package/html-react-parser):

```sh
yarn add html-react-parser@1.2.4
```

**Comment by sugizon** [comment]
Thanks！
